ncm-metaconfig/src/main/metaconfig/cgroups/config/group.tt

```diff
@@ -1,3 +1,12 @@
 [% FOREACH pair IN data.pairs -%]
```

Comment:
too much copy paste imho
can you try with
```
[% IF data.perm.defined %]
[%      INCLUDE "metaconfig/cgroups/config/perm.tt" name='perm' data=data.perm -%]
[% END %]
[% FOREACH pair IN data.pairs -%]
[%-      IF pair.key == 'perm'; NEXT; END -%]
[%      INCLUDE "metaconfig/cgroups/config/${pair.key}.tt" name=pair.key data=pair.value -%]
[% END -%]
```

Reply:
Tested with the refactored code and the output looks correct and test ran successfully. Committed and pushed.